client/state/stats/subscribers/actions.js

```diff
@@ -0,0 +1,23 @@
+import {
+	STATS_SUBSCRIBERS_RECEIVE,
+	STATS_SUBSCRIBERS_REQUEST,
+	STATS_SUBSCRIBERS_REQUEST_SUCCESS,
+	STATS_SUBSCRIBERS_REQUEST_FAILURE,
+} from 'calypso/state/action-types';
+import 'calypso/state/stats/init';
+
+export function requestStatSubscribers( siteId ) {
```

Comment:
Another question is, if we are not going to cross reference the data, would it be easier just use `react-query` meaning we could skip Redux all together?

Reply:
I was also wondering about skipping Redux, but using it gives us more flexibility and performance. I think it helps when navigating to another page, like a summary page and coming back or swapping periods, and most likely also when handling multiple websites. It also aligns with the way we handle other Stats data.
If we decide to extend the section with other modules for subscribers, such as a world map, it will probably be easier to handle compared to passing props.

It would work both ways, with or without Redux, so it's not obvious what to pick, and I'm open to discussion.

Reply:
>  I think it helps when navigating to another page, like a summary page and coming back or swapping periods, and most likely also when handling multiple websites

These could be handled by the `queryKey`, also the results could be cached for some time (default 5 min) during the lifetime of the app. So IMHO, if we are not yet to use subscribers data  with for example visitor data (cross reference in several places), then `react-query` would work beautifully.

> It also aligns with the way we handle other Stats data.
If we decide to extend the section with other modules for subscribers, such as a world map, it will probably be easier to handle compared to passing props.

This makes sense tho. However I guess it doesn't hurt if we only start with `react-query`, because for one, it does the job; for two, we can always add the states for Redux later when we need them easily. 😄 

I'm new to `react-query` approach too and happy to discuss more.

Reply:
We can get rid of a lot of tricky stuff, for example:

https://github.com/Automattic/wp-calypso/blob/6e6636e9633456a68032d28fafd6d734e08db379/client/my-sites/stats/stats-chart-tabs/index.jsx#L85

https://github.com/Automattic/wp-calypso/blob/6e6636e9633456a68032d28fafd6d734e08db379/client/my-sites/stats/stats-chart-tabs/index.jsx#L160

https://github.com/Automattic/wp-calypso/blob/6e6636e9633456a68032d28fafd6d734e08db379/client/my-sites/stats/stats-chart-tabs/index.jsx#L62

Reply:
Taking Redux out would simplify the implementation; simplicity is always better.
I agree that we can always add it if we need to and we've got reference code in previous commits.

Reply:
@annacmc: Let's go ahead and remove the Redux implementation introduced by this PR.

Reply:
I pushed a commit removing all Redux elements.